modules/investigating-kernel-crashes.adoc

```diff
@@ -65,6 +65,82 @@ The `kdump` service is intended to be enabled per node to debug kernel problems.
 
 If you are aware of the downsides and trade-offs of having the `kdump` service enabled, it is possible to enable `kdump` in a cluster-wide fashion. Although machine-specific machine configs are not yet supported, you can perform the previous steps through a `systemd` unit in a `MachineConfig` object on day-1 and have kdump enabled on all nodes in the cluster. You can create a `MachineConfig` object and inject that object into the set of manifest files used by Ignition during cluster setup. See "Customizing nodes" in the _Installing -> Installation configuration_ section for more information and examples on how to use Ignition configs.
 
+.Procedure
+
+You can create a `MachineConfig` object for cluster-wide configuration as follows.
+
+. Optional: Create a content of the `/etc/kdump.conf` file and encode it as base64 if you change the configuration from the default:
```

Comment:
Nit: this sounds slightly awkward to read in my opinion, perhaps
```
If you change the `/etc/kdump.conf` configuration from the default, you can encode it into base64 format to include its content in your `MachineConfig` object:
```

Reply:
Done. Thanks!